packages/dtslint/src/util.ts

```diff
@@ -1,9 +1,14 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
 import assert = require("assert");
 import { pathExists, readFile } from "fs-extra";
 import { basename, dirname, join } from "path";
 import stripJsonComments = require("strip-json-comments");
 import * as ts from "typescript";
 
+export const createRule = ESLintUtils.RuleCreator(
+  (name) => `https://github.com/microsoft/DefinitelyTyped-tools/tree/master/packages/dtslint/src/rules/${name}.ts`
```

Comment:
do we seriously have prettier set to require parens around single parameters? I need to change that.

Reply:
I'm just glad you're using Prettier in the first place 😄